code/__defines/chemistry.dm

```diff
@@ -18,8 +18,7 @@
 #define REAGENTS_PER_SHEET 20
 #define MAX_PILL_SPRITE 20 //max icon state of the pill sprites
 
-#define REAGENTS_OVERDOSE 30
-
+#define REAGENTS_OVERDOSE 20 //Default overdose reduced from 30 to 20. 30u is hard for even 2 poorly communicating doctors to hit; 20u is hard for 1 doctor to hit, but easier for 2 poorly communicating doctors to hit.
```

Comment:
I commented on the thread, but you should look to see how overdoses happen. If I'm not mistaken they're based on metabolized units instead of units present in the body, and if this is going through, you probably want it to be the latter.

Reply:
Mentioned on the thread that I liked the idea. Instead of making overdoses based solely on having a large volume of the chemical in the blood, I've made it a mix of both metabolism and volume threshold overdoses. Now you need to have a volume of the chemical greater than the overdose threshold, _plus_ have metabolised a small amount of the chemical which should mean overdoses aren't **instant**, giving doctors a fair amount of time to rectify their mistake. Thoughts?

Reply:
Please reopen this if it's still a problem.